api/scripts/certification/get-cpf-import-status-from-xml.js

```diff
@@ -16,9 +17,8 @@ async function main() {
 (async () => {
   if (isLaunchedFromCommandLine) {
     try {
-      await main();
+      await executeAndLogScript({ processArgvs: process.argv, scriptFnc: main });
```

Comment:
le log est un détail de l'exécution, je propose ce renommage
```suggestion
      await executeScript({ processArgvs: process.argv, scriptFnc: main });
```